.github/SECURITY.md

```diff
@@ -0,0 +1,7 @@
+# Security policy
+
+Only the latest published release and the HEAD of the default branch are supported.
+
+## Reporting a vulnerability
+
+Please report suspected security vulnerabilities to AlphaKretin#7990 via direct message on Discord or email. We will acknowledge receipt of your report as soon as possible and let you know the next steps.
```

Comment:
I'm not sure how to concisely convey this information, but if possible I'd like to specify that I won't accept friend requests but that my DMs are open if you share a server with me, and of course you can find me on the support server but don't post vulns publicly there. As for email, I have an address mostly dedicated to this sort of thing even though it's not the same one attached to my GitHub, so we should be able to use that. The only issue is I don't know how often I'll check it. It's rigged up to my gmail notifier, but sometimes that disconnects from all my addresses and I don't realise for a while.